src/components/DropMenu/DropMenu.jsx

```diff
@@ -504,6 +513,7 @@ const DropMenu = createClass({
 				'&-direction-up': isExpanded && direction === 'up',
 			}, className)} style={style}>
 				<ContextMenu
+					{...contextMenuProps}
```

Comment:
I couldn't really move it down since it contains the default props, I guess we can move it if it's needed in the future :)